internal/api/genapi/input.go

```diff
@@ -223,6 +224,22 @@ var inputStructs = []*structInfo{
 		createResponseTypes: []string{CreateResponseType, ReadResponseType, UpdateResponseType, DeleteResponseType, ListResponseType},
 		recursiveListing:    true,
 	},
+	// AppToken related resources
+	{
+		inProto:     &apptokens.AppToken{},
+		outFile:     "apptokens/apptoken.gen.go",
+		skipOptions: true,
+	},
+	{
+		inProto:     &apptokens.Grant{},
+		outFile:     "apptokens/grant.gen.go",
+		skipOptions: true,
+	},
+	{
+		inProto:     &apptokens.GrantJson{},
+		outFile:     "apptokens/grant_json.gen.go",
+		skipOptions: true,
+	},
```

Comment:
What is the configuration used for?

Reply:
It's used by internal/api/genapi/main.go via the Makefile in that directory to
generate Go structs which are used by boundary's generated API client to represent
resources when interacting with the API.